src/seedu/addressbook/Main.java

```diff
@@ -103,16 +105,19 @@ private void recordResult(CommandResult result) {
      * @param command user command
      * @return result of the command
      */
-    private CommandResult executeCommand(Command command)  {
+    private CommandResult executeCommand(Command command) throws StorageFileDeletedException {
         try {
             command.setData(addressBook, lastShownList);
             CommandResult result = command.execute();
-            storage.save(addressBook);
+            storage.save(addressBook, true);
```

Comment:
this one is a little confusing. why do you want to modify the default way of handling the save method?